R/utils.R

```diff
@@ -170,3 +170,11 @@ check_connect_version <- function(using_version, minimum_tested_version = "1.8.8
   }
   invisible()
 }
+
+token_hex <- function(n) {
+  raw_to_hex <- function(raw) {
+    paste(as.character(raw), collapse = "")
+  }
+  raw <- as.raw(sample(0:255, n, replace = TRUE))
+  raw_to_hex(raw)
+}
```

Comment:
Does `raw_to_hex` need to exist as a separate function?

```suggestion
token_hex <- function(n) {
  raw <- as.raw(sample(0:255, n, replace = TRUE))
  paste(as.character(raw), collapse = "")
}
```